store/tikv/test_probe.go

```diff
@@ -34,6 +36,43 @@ func (s StoreProbe) NewLockResolver() LockResolverProbe {
 	return LockResolverProbe{LockResolver: newLockResolver(s.KVStore)}
 }
 
+// GetTimestampWithRetry returns latest timestamp.
+func (s StoreProbe) GetTimestampWithRetry(bo *Backoffer, scope string) (uint64, error) {
```

Comment:
Where do we need this function? Or it is just prepareing for the next PRs?

Reply:
Yes. For later PR.